main.yml

```diff
@@ -596,6 +596,7 @@ openshift:
   deployment_repo: https://github.com/RedHatInsights/uhc-portal-frontend-deploy.git
   productId: Red Hat Openshift Cluster Manager
   frontend:
+    module: openshift#./RootApp
```

Comment:
Because `openshift` app is spread accross multiple pages we have to add full `module` description.
```YML
  module:
    appName: openshift
    scope: openshift
    module: ./RootApp
    group: openshift
```